src/coffee/stockimport.coffee

```diff
@@ -12,12 +12,13 @@ xmlHelpers = require './xmlhelpers'
 class StockImport
 
   constructor: (@logger, options = {}) ->
-    options = _.defaults options, {user_agent: 'sphere-stock-import'}
+    options = _.defaults options, {user_agent: 'sphere-stock-import', max409Retries: 10}
```

Comment:
Is this amount supposed to be fixed at 10, or could it be user specified? If it should be user-specified, there's no way (yet) for users to know of this change, and if it should be fixed, why are we not just setting it on L21?